rosmon_core/src/fmt_no_throw.h

```diff
@@ -0,0 +1,27 @@
+#include <fmt/format.h>
+
+#include <iostream>
+
+namespace fmtNoThrow {
+
+template <typename S, typename... Args>
+inline FMT_ENABLE_IF_T(fmt::v5::internal::is_string<S>::value, void)
```

Comment:
I have fmt::v6 and so does the robots. I don't think we should explicitly call the version here. 
FMT_ENABLE_IF_T is a macro that is equivalent to http://www.cplusplus.com/reference/type_traits/enable_if/
From my understanding, if the template arguement `S` is not a fmt string type then during compile time this function will throw an error.
So you can drop this conditional check here as this condition is validated by the `fmt::print` function anyway. Just use, 
```
template <typename S, typename... Args>
inline void print(const S &format_str, const Args &... args) {
    try {
        fmt::print(format_str, args...);
    } catch (const std::exception& e) {
        std::cerr << e.what();
    }
}
```
The same changes for the function below too.